bin-src/lib/getOptions.ts

```diff
@@ -32,7 +27,8 @@ const undefinedIfEmpty = <T>(array: T[]) => {
 export default function getOptions({ argv, env, flags, log, packageJson }: Context): Options {
   const fromCI = !!flags.ci || !!process.env.CI;
   const [patchHeadRef, patchBaseRef] = (flags.patchBuild || '').split('...').filter(Boolean);
-  const [branchName, ownerName] = (flags.branchName || '').split(':').reverse();
+  const [branchName, branchOwner] = (flags.branchName || '').split(':').reverse();
+  const [repositoryOwner] = flags.repositorySlug ? flags.repositorySlug.split('/') : [];
```

Comment:
Can we write a test for this?

Reply:
and maybe how it relates to which one takes precedence on line 67

Reply:
Done